templates/documentation/indicators.html

```diff
@@ -0,0 +1,148 @@
+{% extends 'base.html' %} {% block content %}
+<h2>Climate Indicators</h2>
+
+<p>
+  These endpoints provide access to era-based summaries of the NCAR 12km climate indicators.
+  The eras for this dataset are historical (1988&ndash;2017), mid&ndash;century (2040&ndash;2069),
+  and late&ndash;century (2070&ndash;2099).
+</p>
+
+<p>
+  For more detailed information, see the links to academic references and
+  source datasets included at the bottom of this page.
+</p>
+
+<h3>Service endpoints</h3>
+
+<h4>All climate indicator variables (point query)</h4>
+
+<p>
+  Query min-mean-max climate indicator variables for all models and scenarios.
+</p>
+
+<table class="endpoints">
+  <thead>
+    <tr>
+      <th class="endpoint-label">Endpoint</th>
+      <th class="endpoint-url">Example URL</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Era-based min-mean-max for all climate indicator variables at a given latitude and longitude.</td>
+      <td>
+        <a href="/indicators/base/point/61.5/-147">/indicators/base/point/61.5/-147</a>
+      </td>
+    </tr>
+    <td>Era-based min-mean-max for all climate indicator variables for a specific
+      <a href="/places/all">area of interest polygon ID</a> and aggregate by mean.
+    </td>
+    <td>
+      <a href="/indicators/base/area/61.5/-147">/indicators/base/area/61.5/-147</a>
```

Comment:
```suggestion
      <a href="/indicators/base/area/61.5/-147">/indicators/base/area/GMU7</a>
```
map area example to an area, not a lat-lon